packages/malloy/src/dialect/postgres.ts

```diff
@@ -266,6 +266,10 @@ export class PostgresDialect extends Dialect {
     throw new Error("Not implemented Yet");
   }
 
+  sqlNow(): Expr {
+    return mkExpr`CURRENT_TIMESTAMP`;
```

Comment:
I think this was broken for Postgres too? (We can use either `CURRENT_TIMESTAMP` or `now()`, but `CURRENT_TIMESTAMP()` throws.)